cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/DefaultStore.java

```diff
@@ -85,10 +90,16 @@
  * Implementation of the Store that ultimately places data into MetaDataTable.
  */
 public class DefaultStore implements Store {
-  private static final Logger LOG = LoggerFactory.getLogger(DefaultStore.class);
+
   public static final String APP_META_TABLE = "app.meta";
+  public static final String WORKFLOW_STATS_TABLE = "workflow.stats";
```

Comment:
ok. sounds good. are the reasons documented in your design doc? May want to add a note in a comment here as well, so its easier for people to maintain in future.